capa/features/extractors/ida/insn.py

```diff
@@ -88,6 +99,7 @@ def extract_insn_number_features(f, bb, insn):
         const = capa.features.extractors.ida.helpers.mask_op_val(op)
         if not idaapi.is_mapped(const):
             yield Number(const), insn.ea
+            yield Number(const, arch=get_arch()), insn.ea
```

Comment:
should be pretty lightweight, so not too worried unless profiling shows otherwise. also, we don't have a good global context for caching it right now. probably need to fix this for #218 anyways.